src/balena.sbvr

```diff
@@ -814,8 +816,14 @@ Fact type: device service environment variable has value
 	Necessity: each device service environment variable has exactly one value.
 Fact type: device service environment variable has service
 	Necessity: each device service environment variable has exactly one service.
+	-- This is the default name that will be given to the synonymous form
+	-- once the device service environment variable origin fact type changes
+	Synonymous form: service has device service environment variable
```

Comment:
NIT: Usually we place the `Synonymous form` right below the `Fact type`, and Necessities come right after

Reply:
Done, thanks